src/pages/Owner/NewRestaurant.jsx

```diff
@@ -59,7 +59,7 @@ function NewRestaurant() {
   const handleSubmit = async () => {
     try {
       const res = await sendRequest(
-        `${import.meta.env.VITE_API_URL}restaurant/create`,
+        `${import.meta.env.VITE_API_URL}/restaurant/create`,
```

Comment:
As commented on the earlier commit, `/` is needed to suit env var stored in Live FE server.